GeantFast/src/Output.cc

```diff
@@ -123,19 +117,19 @@ void FCCOutput::CreateHistograms()
    analysisManager->SetH1YAxisTitle(2, "Entries");
 }
 
-//....oooOO0OOooo........oooOO0OOooo........oooOO0OOooo........oooOO0OOooo......
-
-void FCCOutput::SaveTrack(SaveType aWhatToSave, G4int aPartID,  G4int aPDG,
+void Output::SaveTrack(SaveType aWhatToSave, G4int aPartID,  G4int aPDG,
                           G4ThreeVector aVector, G4double aResolution, G4double aEfficiency, G4double aEnergy)
 {
    const G4Event* event = G4RunManager::GetRunManager()->GetCurrentEvent();
    G4int evNo = event->GetEventID();
    G4AnalysisManager* analysisManager = G4AnalysisManager::Instance();
+   ServiceHandle<IMessageSvc> msgh("MessageSvc","GeantOutput");
+   MsgStream log(&(*msgh), "GeantOutput");
```

Comment:
creating ``MsgStream`` objects isn't cheap (locating the ``IMessageSvc`` isn't either)
either have a ``MsgStream m_msg`` data member, or use of one the ``Gaudi`` base classes which has such a data member. (bonus point: it will be easier to switch to thread-safe messagesvc)